shuffle/cli/api.md

```diff
@@ -0,0 +1,131 @@
+# Shuffle API
+
+## shuffle new
+
+**Usage:**
+
+`shuffle new ~/Desktop/TestCoin`
```

Comment:
What do you think about also adding a generic, like `shuffle new <path/to/project>` and then have `shuffle new ~/Desktop/TestCoin` as a more concrete example?

Reply:
oh that's better!!